### PR TITLE
Create redirects for Telemetry and Serverless

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Redirects for Telemetry after docu migration from components to modules
- Redirects for Serverless after docu migration from components to modules

**Related issue(s)**

### CODE REVIEW

File: docs/_redirects [hunk 1]
```diff
@@ -31,3 +31,18 @@
...
+https://kyma-project.io/#/01-overview/telemetry/telemetry-03-traces.md https://kyma-project.io/#/telemetry-manager/user/03-traces 301
+https://kyma-project.io/#/01-overview/telemetry/telemetry-04-tutorials.md https://kyma-project.io/#/telemetry-manager/user/05-tutorials
+https://kyma-project.io/#/05-technical-reference/00-custom-resources/telemetry-01-logpipeline https://kyma-project.io/#/telemetry-manager/user/resources/02-logpipeline 301
+https://kyma-project.io/#/05-technical-reference/00-custom-resources/telemetry-02-logparser https://kyma-project.io/#/telemetry-manager/user/resources/03-logparser 301
+https://kyma-project.io/#/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline https://kyma-project.io/#/telemetry-manager/user/resources/04-tracepipeline 301
+https://kyma-project.io/#/01-overview/serverless/README https://kyma-project.io/#/serverless-manager/user/README 301
+https://kyma-project.io/#/03-tutorials/00-serverless/*  https://kyma-project.io/#/serverless-manager/user/tutorials/README 301
+https://kyma-project.io/#/04-operation-guides/troubleshooting/seerverless/* https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/README 301
```
Comment (a-thaler) [inline docs/_redirects:44]:
```suggestion
https://kyma-project.io/#/04-operation-guides/troubleshooting/serverless/* https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/README 301
```
?

Comment (grego952) [inline docs/_redirects:44]:
```suggestion
https://kyma-project.io/#/04-operation-guides/troubleshooting/serverless/* https://kyma-project.io/#/serverless-manager/user/troubleshooting-guides/README 301
```

File: docs/_redirects [hunk 2]
```diff
@@ -31,3 +31,24 @@
 /docs/kyma/latest/04-operation-guides/* https://kyma-project.io/#/04-operation-guides/README 301
 /docs/kyma/latest/05-technical-reference/* https://kyma-project.io/#/05-technical-reference/README 301
 /docs/kyma/latest/glossary/ https://kyma-project.io/#/glossary 301
+/docs/kyma/latest/01-overview/telemetry/README https://kyma-project.io/#/telemetry-manager/user/README 301
```
Comment (valentinvieriu) [inline docs/_redirects:34]: Not sure if using absolute urls is necessary. Would be better to use relatives urls everywhere, so you can test the PR better in the PR preview.
Consider replacing `https://kyma-project.io/` with `/` in all the redirects

**Review by valentinvieriu — DISMISSED** [review]

**Review by valentinvieriu — DISMISSED** [review]

**Review by valentinvieriu — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

File: docs/05-technical-reference/00-custom-resources/README.md [hunk 3]
```diff
@@ -10,6 +10,5 @@ A custom resource (CR) is an extension to the Kubernetes API which allows you to
 | API Gateway | [APIRule](apix-01-apirule.md) |
 | Eventing | [Subscription](evnt-01-subscription.md), [EventingBackend](evnt-02-eventingbackend.md) |
 | Istio | [Istio](oper-01-istio.md) |
-| Telemetry | [LogPipeline](telemetry-01-logpipeline.md), [LogParser](telemetry-02-logparser.md), [TracePipeline](telemetry-03-tracepipeline.md) |
```
Comment (NHingerl) [inline docs/05-technical-reference/00-custom-resources/README.md:13]: thanks for spotting this!

**Pull request merged**